src/main/java/seedu/address/model/person/TelegramHandle.java

```diff
@@ -0,0 +1,62 @@
+package seedu.address.model.person;
+
+import static java.util.Objects.requireNonNull;
+import static seedu.address.commons.util.AppUtil.checkArgument;
+
+/**
+ * Represents a Telegram handle of a person.
+ * A Telegram handle can only consist of lowercase and uppercase letters, digits, and underscores.
+ * Usernames are case-insensitive.
+ */
+public class TelegramHandle {
+    public static final String MESSAGE_CONSTRAINTS = "Telegram Handles can only use a-z, 0-9 and underscores. "
+            + "Usernames are case-insensitive";
+
+    public static final String VALIDATION_REGEX = "^[a-zA-Z0-9_]{1,}$";
```

Comment:
Might  want to add the @ for telegram, 
OR append it in the Personcard

Reply:
then check if it exist or some logic to prevent 2 @ from printing e.g @@l_dinghan

Reply:
appended into PersonCard